Update certificate-signing-requests.mdx (#18984)
cloudflare/cloudflare-docs: src/content/docs/ssl/edge-certificates/additional-options/certificate-signing-requests.mdx

## Renew a certificate

When you [renew a custom certificate](/ssl/edge-certificates/custom-certificates/renewing/), you can reuse a previously generated CSR.

> Please note that it's not possible to use a different CSR with the same certificate. In this case you will need to upload the certificate as a new certificate, selecting the new CSR.

Reviewer: ```suggestion
Note that it is not possible to use a different CSR with the same certificate. In this case, you must upload the certificate as a new certificate, selecting the new CSR.
```
